tdvt/tdvt/metadata/priority.csv

```diff
@@ -0,0 +1,327 @@
+Test Name Categorized	Priority
+agg.avg	Required
+agg.count	Required
+agg.countd	Required
+agg.max	Required
+agg.min	Required
+agg.stddev	Required
+agg.sum	Required
+agg.var	Required
+BUGS.B340	Required
+BUGS.B1713	Required
+BUGS.B8888	Recommended
+BUGS.B14080	Required
+BUGS.B24394	Required
+BUGS.B26728	Required
+BUGS.B27875-asc-nulls-first	Recommended
+BUGS.B27875-desc-nulls-last	Recommended
+BUGS.B59740	Required
+BUGS.B530764	Required
+BUGS.B641638	Recommended
+BUGS.TFS660780	Required
+calcs_data	Required
+calcs.union	Required for Relational
+cast.float	Required
+cast.float.string	Required
+cast.int	Required
+cast.int.nulls	Required
+cast.str	Required
+cast.str.date	Required
+cast.str.str	Required
+date.B639952	Required
+date.cast.date	Required
+date.cast.datetime	Required
+date.cast.datetime.630831	Smoke
+date.cast.float	Recommended
+date.cast.float.extended	Recommended
+date.cast.int.extended	Recommended
+date.cast.num_to_date	Recommended
+date.cast.str	Required
+date.dateadd.day	Required
+date.dateadd.dayofyear	Required
+date.dateadd.hour	Required
+date.dateadd.iso-week	Recommended
+date.dateadd.iso-weekday	Recommended
+date.dateadd.minute	Required
+date.dateadd.month	Required
+date.dateadd.nulls	Recommended
+date.dateadd.quarter	Required
+date.dateadd.second	Required
+date.dateadd.week	Required
+date.dateadd.weekday	Required
+date.dateadd.year	Required
+date.datediff.day	Required
+date.datediff.dayofyear	Required
+date.datediff.hour	Required
+date.datediff.iso-week	Recommended
+date.datediff.iso-weekday	Recommended
+date.datediff.minute	Required
+date.datediff.month	Required
+date.datediff.nulls	Recommended
```

Comment:
I think assuming the data source supports nulls with date/datetime then I think we need something like "Required if supported"

Reply:
Maybe I'll change "Required for relational" to a more generic "Required if supported".